src/main/controllers/SjpPressListController.ts

```diff
@@ -32,7 +32,7 @@ export default class SjpPressListController {
         sjpData: manipulatedData,
         publishedDateTime: publishedDate,
         publishedTime: publishedTime,
-        contactDate: moment.utc(Date.parse(metaData['contentDate'])).format('D MMMM YYYY'),
+        contactDate: DateTime.fromISO(metaData['contentDate'], {zone: 'utc'}).toFormat('d MMMM yyyy'),
```

Comment:
I've noticed we're using zone: 'utc' here and zone: 'europe/london' in some of the other lists. We should use the same accross all lists that converts the UTC to BST

Reply:
Just looking further, would the helperService.contentDateInUtcTime work for all lists? Potentially could switch to that

Reply:
As discussed, I added Welsh to all the lists.